javagen/src/main/java/com/azure/autorest/mapper/ServiceClientMapper.java

```diff
@@ -54,8 +54,6 @@ public static ServiceClientMapper getInstance() {
 
```

Comment:
this class is mostly refactor, so that methods can be reused by subclass `CadlServiceClientMapper`